plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsConnectorSmokeTest.java

```diff
@@ -89,30 +88,26 @@ DockerizedDataLake createDockerizedDataLake()
         hadoopCoreSiteXmlTempFile.toFile().deleteOnExit();
         Files.write(hadoopCoreSiteXmlTempFile, abfsSpecificCoreSiteXmlContent.getBytes(UTF_8));
 
-        return new DockerizedDataLake(
-                getHadoopBaseImage(),
-                ImmutableMap.of(),
-                ImmutableMap.of(hadoopCoreSiteXmlTempFile.normalize().toAbsolutePath().toString(), "/etc/hadoop/conf/core-site.xml"));
+        HiveMinioDataLake hiveMinioDataLake = new HiveMinioDataLake(
+                bucketName,
+                ImmutableMap.of(hadoopCoreSiteXmlTempFile.normalize().toAbsolutePath().toString(), "/etc/hadoop/conf/core-site.xml"),
+                "ghcr.io/trinodb/testing/hdp3.1-hive");
+        hiveMinioDataLake.start();
+        return hiveMinioDataLake;
     }
 
     @Override
     QueryRunner createDeltaLakeQueryRunner(Map<String, String> connectorProperties)
             throws Exception
     {
-        return createAbfsDeltaLakeQueryRunner(DELTA_CATALOG, SCHEMA, ImmutableMap.of(), connectorProperties, dockerizedDataLake.getTestingHadoop());
-    }
-
-    @Override
-    protected Optional<String> getHadoopBaseImage()
-    {
-        return Optional.of("ghcr.io/trinodb/testing/hdp3.1-hive");
+        return createAbfsDeltaLakeQueryRunner(DELTA_CATALOG, SCHEMA, ImmutableMap.of(), connectorProperties, hiveMinioDataLake.getHiveHadoop());
     }
 
     @AfterClass(alwaysRun = true)
     public void removeTestData()
     {
-        if (adlsDirectory != null && dockerizedDataLake.getTestingHadoop() != null) {
-            dockerizedDataLake.getTestingHadoop().runCommandInContainer("hadoop", "fs", "-rm", "-f", "-r", adlsDirectory);
+        if (adlsDirectory != null && hiveMinioDataLake.getHiveHadoop() != null) {
```

Comment:
i think `hiveMinioDataLake.getHiveHadoop() != null` is not needed. It will never be null when using HibeMinioDataLake